### PR TITLE
Created date-parse.md

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create a resource for the date_parse() function in PHP. Motivation stems from the sustained goal to contribute to docs, learn, and help others.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/date-functions/terms/date-parse/date-parse.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The `date_parse()` is an inbuilt function in PHP which is used to find the detailed information about a specified date.
+
+## Syntax
+
+```pseudo
+date_parse($date);
+```
+
+The `$date` parameter is mandatory. The `date_parse()` function returns an associative array of detailed information for a specified date on success and returns FALSE on failure.
```
Comment (SSwiniarski) [inline content/php/concepts/date-functions/terms/date-parse/date-parse.md:23]:
```suggestion
The `$date` parameter is mandatory. The `date_parse()` function returns an associative array of detailed information for a specified date on success and returns `false` on failure.
```

File: content/php/concepts/date-functions/terms/date-parse/date-parse.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Time'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `date_parse()` is an inbuilt function in PHP which is used to find the detailed information about a specified date.
```
Comment (SSwiniarski) [inline content/php/concepts/date-functions/terms/date-parse/date-parse.md:15]:
```suggestion
The `date_parse()` is an built-in function in PHP which is used to find the detailed information about a specified date.
```

File: content/php/concepts/date-functions/terms/date-parse/date-parse.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## Syntax
+
+```pseudo
+date_parse($date);
+```
+
+The `$date` parameter is mandatory. The `date_parse()` function returns an associative array of detailed information for a specified date on success and returns FALSE on failure.
+
+## Codebyte Example
```
Comment (SSwiniarski) [inline content/php/concepts/date-functions/terms/date-parse/date-parse.md:25]: I'd like to see an additional regular example with an output block that uses the various named parameters, maybe printing them out individually.

At least, I'd like to see this codebyte changed to a regular example with an output block afterwards, like:
````
This results in the following output:

```shell
Array
(
    [year] => 2023
    [month] => 6
    [day] => 27
    [hour] => 11
    [minute] => 27
    [second] => 40
    [fraction] => 0.7
    [warning_count] => 0
    [warnings] => Array
        (
        )
    [error_count] => 0
    [errors] => Array
        (
        )
    [is_localtime] => 
)
```
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@arisdelacruz, I did a first review and left a few comments.

File: content/php/concepts/date-functions/terms/date-parse/date-parse.md (hunk 3 above)
Comment (arisdelacruz) [reply content/php/concepts/date-functions/terms/date-parse/date-parse.md:25]: @SSwiniarski 

noted on this

thanks

Comment (arisdelacruz) [reply content/php/concepts/date-functions/terms/date-parse/date-parse.md]: @SSwiniarski 

i did a pull and a push

thanks

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz @SSwiniarski Looks good. Just corrected one typo. This is ready to merge

**Comment by SSwiniarski** [comment]
@arisdelacruz Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/date-functions/date-parse

image

**Pull request merged**